examples/async-multiplexed.rs

```diff
@@ -0,0 +1,55 @@
+extern crate futures;
+extern crate redis;
+extern crate tokio;
+use futures::{future, prelude::*};
+use redis::{aio::MultiplexedConnection, RedisResult};
+
+fn test_cmd(con: &MultiplexedConnection, i: i32) -> impl Future<Output = RedisResult<()>> + Send {
```

Comment:
This could be an `async` function

Reply:
I'll need some guidance on return values etc.  The compiler wins again.